pandas/tests/frame/methods/test_describe.py

```diff
@@ -298,3 +298,14 @@ def test_describe_percentiles_integer_idx(self):
             ],
         )
         tm.assert_frame_equal(result, expected)
+
+
+def test_unhashable_dict():
+    # GH#32409
+    result = pd.DataFrame(
```

Comment:
Thanks @rivera-fernando 

What's the difference between `result` and expected? Can you construct `result` using `.describe` (as in the original issue)?